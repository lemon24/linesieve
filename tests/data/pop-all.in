--section '(\S+):$'
push o
  sub 1 2
  push t
    sub 2 3
pop --all
sub '(\d)' '\1\1'
---
one:
1
two:
1
three:
1
four:
1
fifty:
1
fiftyfive:
2
