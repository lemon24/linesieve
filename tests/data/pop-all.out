one
33
two
33
three
11
four
33
fifty
11
fiftyfive
33
